internal/common/simulation/README.md

```diff
@@ -0,0 +1,10 @@
+## Numerical Simulation
+
+### ODEs
+
+Differential equations are extremely useful for modelling evolving quantities such as animal populations that change in response to some stimulus. This package provides a basic ODE (ordinary DE) solver interface using the 4th order Runge-Kutta method (**RK4**). Read more on this [here](https://en.wikipedia.org/wiki/Runge–Kutta_methods#The_Runge–Kutta_method).
```

Comment:
Still in a bit of disbelief that 1. There's no ODE solver sitting around to be used off the shelf, 2. The madlad wrote it himself. Good job @JamesTev